IceNet/polar-modelling-icenet.ipynb

```diff
@@ -0,0 +1,1411 @@
+{
```

Comment:
<p>"use a sample data" =&gt; "use a small sample of the data"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/PangeoExamples/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
<p>done, thanks!</p>